services/discovery/src/app.py

```diff
@@ -112,8 +115,14 @@ def coeffects_s3(world):
 
 @coeffect('cur')
 def coeffects_cur(world):
-    response = cur.describe_report_definitions()
-    return keyfilter(lambda x: x in {'ReportDefinitions'}, response)
+    try:
+        return {
+            'is_master_payer': True,
+            'report_definitions': cur.describe_report_definitions().get('ReportDefinitions', []),
```

Comment:
As long as `cur.describe_report_definitions` doesn't throw a `ClientError`, `is_master_payer` should be `True`